schedule/yam/agama_unattended_lvm.yaml

```diff
@@ -0,0 +1,10 @@
+---
```

Comment:
agama_lvm_unattended (so we can see together in openQA both interactive and unattended).

Reply:
updated, thanks for review.